spec/models/miq_provision_request_template_spec.rb

```diff
@@ -153,6 +153,14 @@
     end
   end
 
+  describe "post_create" do
+    it 'sets description' do
+      provision_request_template.post_create(true)
```

Comment:
What do you think about adding an `expect` to ensure that `MiqAeEngine` **does not** receive a call to `resolve_automation_object`.  Which I believe is a big part of what we are trying to resolve with this change.

Outside of that I'm good.